torchgen/gen_backend_stubs.py

```diff
@@ -140,6 +140,7 @@ def create_backend_index(
             dispatch_key=dispatch_key,
             use_out_as_primary=use_out_as_primary,
             external=True,
+            symint=True,  # TODO: make this configurable
```

Comment:
hopefully I can answer this question myself as I keep reading:

If you set this to false, would that technically be less BC-breaking for e.g. XLA? (not that it matters much in this case since you pretty much have to do an XLA-side patch to remove the `empty.SymInt` overload).

From my understanding, setting this to `True` will tell the codegen to generate kernel stubs for the backend that take in `SymIntArrayRef`/`SymInt`, and setting it to `False` will tell the codegen to generate stubs taking in `IntArrayRef`/`int64_t`.

Reply:
If that's true, I wonder if `False` is a better default, since e.g. Intel is using this codegen, and probably doesn't plan to change any of their kernels to

Reply:
I'm overfit for XLA, which needs to get the SymInt variant. Also, you can always workaround if it's SymInt, but not if it's int. Toggleable is right.